scss/typography/_base.scss

```diff
@@ -441,3 +433,26 @@ $abbr-underline: 1px dotted $black !default;
     }
   }
 }
+
+/// Calculates the header font-size based on breakpoints.
+///
+/// @param {String} the size of the header to output.
+@mixin typography-header-font-size (
+  $requested-size
+) {
```

Comment:
@kball I think `$requested-size` is intended to generated the default font-size, like in `/scss/_global.scss`. I don't see other purposes. <span class="emoji-outer emoji-sizer">

@Shebs72 Can you give us an exemple where `requested-size` would be useful ? Otherwise, please remove it.